app/src/main/java/org/mozilla/fenix/tabtray/FenixTabsAdapter.kt

```diff
@@ -104,7 +105,6 @@ class FenixTabsAdapter(
     private fun showCheckedIfSelected(tab: Tab, view: View) {
         val shouldBeChecked =
             mode is TabTrayDialogFragmentState.Mode.MultiSelect && selectedItems.contains(tab)
-        view.checkmark.isVisible = shouldBeChecked
```

Comment:
does the checkmark visibility not need to be set anymore to match `shouldBeChecked` with this change? I don't see anything else updating its checked state. Was it unnecessary in the first place?

Reply:
Previously the layout tree looked like:
```
- parent
  |- image
  |- mask
  |- checkmark
```

Now it looks like:
```
- parent
  |- image
  |- mask
     |- checkmark
```

Since the checkmark's parent is hidden we don't need to hide the checkmark itself.